onboard-server/src/main/java/no/vegvesen/ixn/serviceprovider/OnboardRestController.java

```diff
@@ -116,7 +108,7 @@ public void deleteCapability(@PathVariable String serviceProviderName, @PathVari
 	public GetCapabilityResponse getServiceProviderCapability(@PathVariable String serviceProviderName, @PathVariable String capabilityId) {
 		OnboardMDCUtil.setLogVariables(selfService.getNodeProviderName(), serviceProviderName);
 		this.certService.checkIfCommonNameMatchesNameInApiObject(serviceProviderName);
-		ServiceProvider serviceProvider = serviceProviderRepository.findByName(serviceProviderName);
+		ServiceProvider serviceProvider = getOrCreateServiceProvider(serviceProviderName);
 		Capability capability = serviceProvider.getCapabilities().getCapabilities().stream().filter(c ->
```

Comment:
Possibly do the same when deleting a Capability?

Reply:
?